pallets/loans-ref/src/lib.rs

```diff
@@ -473,7 +480,7 @@ pub mod pallet {
 		/// - Write off by admin with percentage 0.5 and penalty 0.2
 		/// - Time passes and the policy can be applied.
 		/// - Write of with a policy that says: percentage 0.3, penaly 0.4
-		/// - The loan is written off with the maximum between the policy and the current state:
+		/// - The loan is written off with the maximum between the policy and the current rule:
 		///   percentage 0.5, penaly 0.4
```

Comment:
Nit
```suggestion
		///   percentage 0.5, penalty 0.4
```

Reply:
Realizing that only means you did a deep review, thanks!!